backend/src/main/java/de/neuefische/koheis/backend/chractercards/CharacterCardWithoutId.java

```diff
@@ -0,0 +1,12 @@
+package de.neuefische.koheis.backend.chractercards;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
same here, `@Data` in combination with `private final` should be enough here

Reply:
👍